api/server/router/local/image.go

```diff
@@ -63,12 +63,11 @@ func (s *router) postCommit(ctx context.Context, w http.ResponseWriter, r *http.
 		Config:  c,
 	}
 
-	container, err := s.daemon.Get(cname)
-	if err != nil {
+	if !s.daemon.Exists(cname) {
 		return err
```

Comment:
no, that's wrong. I need to take another look. Should probably be a derr.containernotexists or somesuch.